apps/web/src/views/V3Info/data/token/priceData.ts

```diff
@@ -264,17 +268,23 @@ export async function fetchPairPriceChartTokenData(
     }
 
     const formattedHistory = data.map((d) => {
+      const high = parseFloat(d.high)
+      const low = parseFloat(d.low)
+      if (high >= maxPrice) maxPrice = high
```

Comment:
If no max set max if has max check if higher set max

Reply:
I had set the default maxPrice as 0